monitoring/README.md

```diff
@@ -27,5 +27,4 @@ This will make sure `prometheus` can scrape the metrics from `eigenda` node.
 
 
 #### Useful Dashboards
-We also provide a set of useful Grafana dashboards which would be useful for monitoring the EigenDA node. You can find them [here](../dashboards).
-Once you have Grafana setup, feel free to import the dashboards.
\ No newline at end of file
+We also provide a set of useful Grafana dashboards that will be automatically imported when you start the monitoring stack.
```

Comment:
Reworded proposal (in line with previous comment) and avoiding use of redundant terms such as "also":

> EigenDA offers a set of Grafana dashboards that are automatically imported when initializing the monitoring stack